roles/cobbler_profile/tasks/import_distro_iso.yml

```diff
@@ -42,6 +42,16 @@
   register: import
   when: mount is defined and mount is changed
 
+- name: Rename the distro if the arch is arm
```

Comment:
Can you just add a comment explaining why this is necessary please?

Smart workaround!